tcms/rpc/serializer.py

```diff
@@ -170,7 +171,7 @@ def _serialize_names(row, values_fields_mapping):
     return new_serialized_data
 
 
-class QuerySetBasedXMLRPCSerializer(Serializer):
+class QuerySetBasedRPCSerializer(Serializer):
     """XMLRPC serializer specific for TestPlan
```

Comment:
Doc-string not updated. Use "Serializer for ..." like in some of the classes below.